packages/plugin-ext/src/main/browser/dialogs/style/modal-notification.css

```diff
@@ -60,22 +60,22 @@
 .modal-Notification .text {
     order: 2;
     display: inline-block;
-    max-height: calc(100vh - 100px);
-    max-width: calc(100vw - 100px);
     -webkit-user-select: text;
     -moz-user-select: text;
     -ms-user-select: text;
     user-select: text;
     align-self: center;
     flex: 1 100%;
-    height: 100%;
+    max-height: min(66vh, 600px);
```

Comment:
Instead of modifying `height` to `max-height`, can you remove it and editor `max-height` [above](https://github.com/eclipse-theia/theia/pull/8080/files#diff-3ff0cf7e13a83017520974a95117399eL63).

Reply:
For both of these I did it this way because I prefer to write my widths/heights like this for readability:
```
min-height:
height:
max-height:
```